src/content/docs/infrastructure/amazon-integrations/connect/connect-aws-new-relic-infrastructure-monitoring.mdx

```diff
@@ -111,4 +111,4 @@ To [uninstall your services completely](/docs/infrastructure/new-relic-infrastru
 
 ## Regional support [#availability]
 
-China regions are [not supported.](/docs/integrations/amazon-integrations/troubleshooting/invalid-principal-error-unsupported-aws-regions)
+China regions are [not supported](/docs/integrations/amazon-integrations/troubleshooting/invalid-principal-error-unsupported-aws-regions).
```

Comment:
As vale says, should be a contraction. Also since this line is being changed in this PR already, does "China regions" read weirdly to you? It's clear and makes sense in what it's trying to convey but I feel like it's kind of wonky, like it should be "Chinese regions" or something. Just some food for thought.